stdlib/json.gr

```diff
@@ -2069,3 +2083,242 @@ provide let parse: (str: String) => Result<Json, JsonParseError> = (str: String)
     }
   }
 }
+
+/**
+ * Extract the value of a JSON boolean value, or `Err` if the JSON value is not
```

Comment:
I feel like this feels a bit wordy what about something like `Extracts the value of a JSON boolean.`, we already explain the `Err` part in the returns.

This feedback is bassically for all the functions.